ui/apps/platform/src/Containers/MainPage/MainPage.tsx

```diff
@@ -98,6 +99,7 @@ function MainPage(): ReactElement {
                     hasServiceIdentityWritePermission={hasServiceIdentityWritePermission}
                 />
                 {metadata?.stale && <VersionOutOfDate />}
+                 <DatabaseBanner isApiReachable={serverState && serverState !== 'UNREACHABLE'} />
```

Comment:
style-checks reports an error for extra space at the left of `<DatabaseBanner` directly under `{` above